masterdata/configuration/liquibase/liquibase.xml

```diff
@@ -375,4 +375,51 @@
             INSERT INTO provider (person_id, identifier, creator, date_created, uuid, name) VALUES (@person_id, 'avniintegrator', 1, now(), '36313e2c-0098-11ee-be56-0242ac120002', 'avniintegrator');
         </sql>
     </changeSet>
+
+    <!--Adding RCH ID-->
+    <changeSet id="Adding-RCH-ID-IdentifierType-GOK-157"  author="Pooja">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">select count(*) from patient_identifier_type where name='RCH ID'</sqlCheck>
+        </preConditions>
+        <comment>Adding RCH ID Identifier type</comment>
+        <sql>
+            insert into patient_identifier_type( name, description,creator, uuid, uniqueness_behavior, location_behavior, date_created) Select 'RCH ID','RCH ID identifier type',creator,uuid(),'UNIQUE','NOT_USED',now() from users where username='admin';
+        </sql>
+    </changeSet>
+
+    <!--Adding Nikshay ID-->
+    <changeSet id="Adding-Nikshay-ID-IdentifierType-GOK-157"  author="Pooja">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">select count(*) from patient_identifier_type where name='Nikshay ID'</sqlCheck>
+        </preConditions>
+        <comment>Adding Nikshay ID Identifier type</comment>
+        <sql>
+            insert into patient_identifier_type( name, description,creator, uuid, uniqueness_behavior, location_behavior, date_created) Select 'Nikshay ID','Nikshay ID identifier type',creator,uuid(),'UNIQUE','NOT_USED',now() from users where username='admin';
```

Comment:
Tried this query on my machine, and the insert is not happening it seems. I don't have the latest setup, might be an issue on my end but for me the no insertions took place.
<img width="1676" alt="Screenshot 2023-06-15 at 10 26 01 PM" src="https://github.com/Bahmni-HWC/clinic-config/assets/121226043/7b45b597-356b-4934-bed8-221d70e4770d">

Reply:
`INSERT INTO patient_identifier_type( name, description,creator, uuid, uniqueness_behavior, location_behavior, date_created) SELECT 'RCH ID','RCH ID identifier type',creator,uuid(),'UNIQUE','NOT_USED',now() FROM users WHERE system_id='admin';`

By updating the where clause to query results where the system_id='admin' instead of username='admin' solved the issue for me.
<img width="1676" alt="Screenshot 2023-06-15 at 10 38 27 PM" src="https://github.com/Bahmni-HWC/clinic-config/assets/121226043/eff774ad-7e3f-4e50-b281-7faf065f940a">